### DISCUSSION TITLE
need support to add custom EVM chain on CDN version

### DISCUSSION THREAD

**Discussion by oleksiivinogradov** [WalletConnect/walletconnect-monorepo#3024] [Developer Support]
Hey team. 
I can't understand, why do you kill our time and own business? You stop V1 support and has solution for CDN only few days before stopping.

https://docs.walletconnect.com/2.0/web/web3modal/html/wagmi/installation (with broken example link!!!)

It works, but only for chains in WagmiCoreChains list.

If I would like to add custom chain with custom ID and rpcURL - I can't! I have to use jsonRpcProvider which is not reached.

Why developers have to spend a days own time (i  sure that 1k+ devs has same issue) for you unresponsible steps????

I expect same answer maybe (tell us a version of SDK????) but already do all follow https://docs.walletconnect.com/2.0/web/web3modal/html/wagmi/installation

!!!!!

**Comment by oleksiivinogradov** [comment]
simple solution (add in docs please!!!)
```
const mantleTestnet = {
			id: 5001,
			name: "Mantle Testnet",
			network: "mantleTestnet",
			nativeCurrency: {
				decimals: 18,
				name: "tMNT",
				symbol: "tMNT"
			},
			rpcUrls: {
				default: {
					http: ["https://rpc.testnet.mantle.xyz"],
				},
				public: {
					http: ["https://rpc.testnet.mantle.xyz"]
				}
			},
			blockExplorers: {
				etherscan: {
					name: "mantle Scan",
					url: "https://explorer.testnet.mantle.xyz/"
				},
				default: {
					name: "eos Scan",
					url: "https://explorer.testnet.mantle.xyz/"
				}
			}

		};


		const chains = [aurora, bscTestnet, auroraTestnet, polygon, eos, mantleTestnet]
```

**Comment by crypblizz8** [comment]
Hey there, thanks for reaching out and understand the frustration. We have been telling developers to update for several months so it was planned well in advance. 

That being said, can you tell me where you are stuck with. Custom chains is shown here on [wagmi](https://wagmi.sh/react/providers/configuring-chains) as your code demonstrates. Please let us know how we can help.
